src/content/articles/test-article.json

```diff
@@ -0,0 +1,7 @@
+{
+    "id": "test-article",
```

Comment:
These files are under /src, which is probably not ideal.

I tried /content with no success. I could also see /static, though I'm not sure what ramifications that has.